napari/components/_viewer/model.py

```diff
@@ -59,30 +60,15 @@ def __init__(self, title='napari'):
         self.key_bindings = {}
 
         # TODO: this should be eventually removed!
-        # initialised by QtViewer when it is constructed by the model
-        self._qtviewer = None
+        # attaced by QtViewer when it is constructed by the model
```

Comment:
```suggestion
        # attached by QtViewer when it is constructed by the model
```
also is it necessary to have this on here? if it's `None`, none of the related code should run anyways

Reply:
typo fixed. this is only initialized to `None`, it gets set to be the `scene` inside the `QtViewer`, but we will remove this in the future

Reply:
Yes, but why is it initialized at all? It's going to break regardless; I don't think it belongs here.

Reply:
What do you mean by "break"? It runs fine now. The reason I put it here was that it gets used again below, line 248 `layer._parent = self._scene`. I'm not sure where else that could be

Reply:
ah, ok I didn't notice that. this was more a callback to the initial dims refactor PR which introduced this (and wasn't necessary at the time)